ts/app.ts

```diff
@@ -79,6 +87,28 @@ module MRoot {
             $scope.getAccountName = () => {
                 return $cookies.accountName;
             };
+            
+            $scope.changePassword = (win : kendo.ui.Window)=>{
+                $scope.modelChangePassword = new ChangePasswordModel();
+                win.center();
+                win.open();
+            }
+
+            $scope.funcChangePasswordDone = (win : kendo.ui.Window)=>{
+                var msg = new ApiHeader.APIUpdateAccountMsg();
+                msg.uuid = '36c27e8ff05c4780bf6d2fa65700f22e';

```

Comment:
So how to know that we should call `APIUpdateAccountMsg` or `APIUpdateUserMsg`. And which method is the best to access accountUuid or userUuid, store them in cookies as accountName does?